### ISSUE TITLE
Multiple bugs related to voip calling

### ISSUE THREAD

**Issue by GammaPi** [vector-im/riotX-android#1719]
Version: Element 1.0, F-Droid version
Test Platform: Huawei EMUI 10 (two devices), Computer running Elements Desktop 1.7.1

Bug1: When receiving calls, selected ringtone is played with default sound (on the phone).
Bug2: Calling window won't pop up, and only shows in notification panel (on the phone).
Bug3: If I open the app during an incoming call, there's only a hangup button inside that room (no  incoming-call window popup, and no accept button inside room) (on the phone)
Bug4: Login same account in phone1 and computer and call phone2. Then, reject the call on the computer. Incoming call notification still persists on phone2. If you click on that notification and open incoming-call window, accept/hangup button will still be shown(ringtone stops though). At this time, if you click one of these, these two buttons will disappear and the  incoming-call window cannot be exited.

**Comment by Wandelstock** [comment]
Bug 7. If call ended on other devices (of the same recieving account) there is still the accept or reject button of the call (on a full lit screen). (Android)

This must be ended manually, by each device

**Comment by GammaPi** [comment]
> Bug 7. If call ended on other devices (of the same recieving account) there is still the accept or reject button of the call (on a full lit screen). (Android)
> 
> This must be ended manually, by each device

That's bug4

**Comment by GammaPi** [comment]
Additional info to Bug3:
Incoming-call window will pop up if the screen is off. 
However,
- if the screen is on incoming-call window won't pop up and will.
- if the screen and I open the app, the incoming-call window still won't pop up.

**Comment by Wandelstock** [comment]
Yup, you are right, sorry. Do you want me to delete it?

**Comment by GammaPi** [comment]
> Yup, you are right, sorry. Do you want me to delete it?

Please don't, maybe it will help people quickly understand bug4, since my description is lengthy and wordy.

**Comment by rajil** [comment]
Here is another one, the default ringtone/sound in Call Notification is set to silent instead of ring.ogg

**Comment by rajil** [comment]
Make it 9, proximity sensor is not used to turn off the screen when brought near the ear.

**Comment by rayman93** [comment]
> Make it 9, proximity sensor is not used to turn off the screen when brought near the ear.

I've opened a separate issue #1735 for that.

**Comment by Rares95** [comment]
https://github.com/vector-im/riotX-android/issues/1790 Bug3

**Comment by Rares95** [comment]
Consider adding https://github.com/vector-im/riotX-android/issues/1788 or https://github.com/vector-im/riotX-android/issues/1688

**Comment by abdullasharapov** [comment]
Bug10: While calling on the phone, it plays call ringtone on caller's phone through the earpiece, but you need it through the loudspeaker

**Comment by MeiRos** [comment]
Customized ringtone still plays once.
Element 1.1.6

**Comment by nat1on** [comment]
bug7: ringtone still plays only once on element android 1.1.9
My experience is that by default, element plays the same ringtone as configured for phone. But the settings are very confusing. After I enter Settings > Notifications > Configure Call Notifications I see a scree which shows Alerting as the chosen option, but under advanced, Sound is set to none and Vibrate is turned off. Still an incoming call plays the default phone ringtone.

OS: Android 10, security patch level May 1, 2021 on Nokia 9 PureView TA-1087
